pages/content/amp-dev/pixi.html

```diff
@@ -0,0 +1,34 @@
+---
+$title: Pixi
+$view: /views/custom.j2
+description: The new pixi tool.
+---
+{% do doc.styles.addCssFile('css/components/templates/pixi.css') %}
+
+<main class="ap--main ap-t-pixi">
+  <section class="ap--stage ap--container-fluid">
+    {% do doc.styles.addCssFile('css/components/organisms/stage.css') %}
+
+    <div class="ap-o-stage">
+      <div class="ap--container-fluid">
+        <div class="ap-o-stage-content">
+          <h1 class="ap-o-stage-content-headline">Web Vitals and<br>AMP Headline</h1>
+        </div>
+        <div class="ap-o-stage-image">
+          {% do doc.icons.useIcon('icons/vitals-header.svg') %}
+          <svg>
+            <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#vitals-header"></use>
+          </svg>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="ap-t-pixi-intro">
+    <p class="ap-t-pixi-intro-text">Web Vitals is an initiative by Google to provide unified guidance for quality signals that are essential to delivering a great user experience on the web.<br><br>Google has provided a number of tools over the
```

Comment:
@nainar can you review this copy?

Reply:
Let's remove Google has provided a number of tools over the .. it isn't adding value. 

Also can we link the phrase Web Vitals to web.dev/vitals?